tasks.py

```diff
@@ -147,7 +148,7 @@ def docker_compose(context, command, **kwargs):
         **kwargs: Passed through to the context.run() call.
     """
     compose_command_tokens = [
-        "docker-compose",
+        "docker-compose" if shutil.which("docker-compose") else "docker compose",
```

Comment:
Consider inverting this - `docker compose` should be preferred, IMHO. See also #2207.

Reply:
True... though running `docker compose --help` as a check at least would return very quickly?

Reply:
I just don't know if the extra complexity required is worth it. Nobody else has reported this and this fix is specific to ltm1.6 core development. Should we mirror 2.x behavior and change it to `docker compose` since we know all of the core developers have that anyway?

Reply:
I think maybe just use `docker compose` since we are using that exclusively in 2.x like Glenn mentioned